src/connector/src/parser/mod.rs

```diff
@@ -1145,6 +1169,8 @@ impl SpecificParserConfig {
         info: &StreamSourceInfo,
         with_properties: &BTreeMap<String, String>,
     ) -> ConnectorResult<Self> {
+        const JSON_SINGLE_BLOB_COLUMN_KEY: &str = "single_blob_column";
```

Comment:
Wait, why this is called "blob" (**Binary** Large Object)?

Reply:
@neverchanje PTAL. I agree `single_jsonb_column` would be better.

Reply:
updated